packages/server/lib/util/print-run.ts

```diff
@@ -629,6 +630,12 @@ export const printCompletedArtifactUpload = <T extends ArtifactUploadResultLike>
     process.stdout.write(`- Failed Uploading ${num}`)
   }
 
+  if (duration) {
+    const durationOut = env.get('CYPRESS_INTERNAL_ENV') === 'test' ? 'Xm, Ys ZZms' : humanTime.short(duration, 2)
```

Comment:
We should really be normalizing this output in the system-tests: https://github.com/cypress-io/cypress/blob/develop/system-tests/lib/normalizeStdout.ts#L112-L113

Reply:
Agreed, however, this pattern is too close to other system tests, so I reverted https://github.com/cypress-io/cypress/pull/28226/commits/1f29082637283f640affe7df4d5354737300727d